### ISSUE TITLE
Users can have a different value for university in the 'members' table and the 'university members' table.

### ISSUE THREAD

**Issue by oshotton** [SWAuk/com_swa#111]
Club management seems to look at value in the members table.

**Comment by addshore** [comment]
Hmm, yeh.
We can probably think of a better way to do this :)
We will have to talk about it! :dancer: 

member table should perhaps not have a uni field.
instead university_member would have an approved field..

**Comment by oshotton** [comment]
Yeah I was thinking something along the same lines (get out of my brain! :P)!

We can discuss when you are home next! :)
